app/e2e/packetPage.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { test, expect, Locator } from "@playwright/test";
+import {
+  getContentLocator,
+  getPacketPageAccordionSection,
+  navigateToFirstPacketGroupLatestPacket,
+  selectPacketPageTab
+} from "./utils";
+
+// Generic packet page tests which will be run against all servers
+test.describe("Packet page", () => {
+  let content: Locator;
+  let packetGroupName, latestPacketId: string;
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto("./");
+    content = await getContentLocator(page);
+    const { packetGroup, packetId } = await navigateToFirstPacketGroupLatestPacket(content);
+    packetGroupName = packetGroup;
+    latestPacketId = packetId;
+  });
+
+  test("can see packet group name and packet id", async () => {
+    await expect(await content.getByRole("heading", { name: packetGroupName, level: 2 })).toBeVisible();
+    await expect(await content.getByText(latestPacketId)).toBeVisible();
+  });
+
+  test.describe("Summary", () => {
+    test("can see Parameters section", async ({ page }) => {
+      const content = await getContentLocator(page);
+      const parametersDiv = await getPacketPageAccordionSection(content, "Parameters");
+      // Section should contain either "None" or at least one parameter pill
+      const noneDiv = await parametersDiv.locator(".italic");
+      if ((await noneDiv.count()) > 0) {
+        await expect(noneDiv).toHaveText("None");
+      } else {
+        await expect(await parametersDiv.locator("div.rounded-md").count()).toBeGreaterThan(0);
+      }
+    });
+
+    test("can see Dependencies section", async ({ page }) => {
+      const content = await getContentLocator(page);
+      const dependenciesDiv = await getPacketPageAccordionSection(content, "Dependencies");
+      // Section should contain either no depencies text, or at least one list item with link
+      const noneDiv = await dependenciesDiv.locator(".italic");
+      if ((await noneDiv.count()) > 0) {
+        await expect(noneDiv).toHaveText("This packet has no dependencies on other packets");
+      } else {
+        const depItems = await dependenciesDiv.getByRole("listitem");
+        await expect(await depItems.count()).toBeGreaterThan(0);
+        for (const li of await depItems.all()) {
+          const link = await li.getByRole("link");
+          await expect(link).toBeEnabled();
+          const linkText = await link.innerHTML();
+          expect((await link.getAttribute("href")).split("/").at(-1)).toBe(linkText);
```

Comment:
why is linkText expected to be part of the href? is it a packet id? if so, would be helpful to rename:
```suggestion
          const packetId = await link.innerHTML();
          expect((await link.getAttribute("href")).split("/").at(-1)).toBe(packetId);
```